pkg/clients/rds/rds_test.go

```diff
@@ -27,3 +29,31 @@ func TestIsErrorNotFound(t *testing.T) {
 		t.Errorf("IsErrorNotFound: want=%v, get=%v", true, isErrorNotFound)
 	}
 }
+
+func TestDBInstanceOperations(t *testing.T) {
```

Comment:
I really like that you're adding some tests in a separate commit to increase test coverage! That said, could you please convert this one to an idiomatic Crossplane-style test that uses `cmp` instead of `assert` per our [contributing guidelines](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md):

> All Crossplane code must be covered by tests. Note that unlike many Kubernetes projects Crossplane does not use gingko tests and will request changes to any PR that uses gingko or any third party testing library, per the common [Go test review comments](https://github.com/golang/go/wiki/TestComments). Crossplane encourages the use of table driven unit tests. The tests of the crossplane-runtime project are representative of the testing style Crossplane encourages; new tests should follow their conventions. Note that when opening a PR your reviewer will expect you to detail how you've tested your work. For all but the smallest changes some manual testing is encouraged in addition to unit tests.

Reply:
Done. I have converted all unit-tests in rds_test.go to table-driven style and double checked the others.